pkg/cmd/admin/policy/create.go

```diff
@@ -0,0 +1,104 @@
+package policy
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	authapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/templates"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const CreateClusterRoleRecommendedName = "create-cluster-role"
+
+var createClusterRoleExample = templates.Examples(`
+	# TODO
+	%[1]s dev --resources a,b,c --actions x,y,z`)
+
+type NewClusterRoleOptions struct {
+	ClusterRoleClient client.ClusterRoleInterface
+
+	Name      string
+	Resources []string
+	Actions   []string
```

Comment:
name the options and flags to match the API object fields? (verbs, resources, api-groups, etc)?